### PR TITLE
ZCS-15077: Fixed global.properties file to add AD fixes.

### PR DESCRIPTION
**Pull request by pallavikhairnar21**
**Fixed global.properties:**
Added 9071 Admin port
Added new AD details
Fixed LDAP user details 

Test case execution result is attached in the https://synacor.atlassian.net/browse/ZCS-15077

### CODE REVIEW

File: conf/global.properties [hunk 1]
```diff
@@ -74,30 +74,30 @@ regex.id=[0-9\-]+
...
 zimbraMessageIdDedupeCacheSize.default=3000
-AD.url=10.139.244.162
+AD.url=129.154.238.213
 AD.port=3268
-AD.domain=zmexch.eng.vmware.com
-AD.userName=administrator
-AD.password=z1mbr4Migration
-AD.passwordChangeCase=z1mbr4MigratiON
-AD.LDAPSearchBase=cn=users,dc=zmexch,dc=eng,dc=vmware,dc=com
+AD.domain=eng.zimbradev.com
```
Comment (nilamraut24) [inline conf/global.properties:79]: Should it be eng.zimbra.com or zimbradev.com instead of eng.zimbradev.com?

**Review by nilamraut24 — COMMENTED** [review]
Line

File: conf/global.properties (hunk 1 above)
Comment (pallavikhairnar21) [reply conf/global.properties:79]: @nilamraut24 New Ad domain is "eng.zimbradev.com" , so it is correctly set there.

Comment (nilamraut24) [reply conf/global.properties:79]: Ok @pallavikhairnar21 got it. Thank you.

**Review by nilamraut24 — APPROVED** [review]
Changes look good to me.

**Review by prashantajari — APPROVED** [review]

**Pull request merged**